pyproject.toml

```diff
@@ -20,9 +20,13 @@ dynamic = ["version", "readme", "dependencies"]
 avalanche = [
     "avalanche_lib==0.3.1",
 ]
+benchmark = [
+    "wild-time-data==0.0.4",
```

Comment:
I am bit puzzled by this dependency because I thought this was a way to get rid of the WildTime dependencies but we are still going to install sklearn etc.

Can we include in Renate the little code we need and rely on the serialized data to decouple it from WildTime?
I am happy to discuss this offline if you want since it's a more complex question.